lux/executor/Executor.py

```diff
@@ -74,7 +74,7 @@ def invert_data_type(self, data_type):
     def compute_data_model(self, data_type):
         data_type_inverted = self.invert_data_type(data_type)
         data_model = {
-            "measure": data_type_inverted["quantitative"],
+            "measure": data_type_inverted["quantitative"] + data_type_inverted["geoshape"],
```

Comment:
In the current system, secondary geo attributes like `state` are still declared as nominal. I wonder if we should be labeling them as geo attributes also. If so this may need to account for non-quantitative geo types?
For now, let's take out the geo data type from measures so that they are not plotting in the correlation tab in the example shown.

Reply:
I see. I may have to take a closer look at this and ask you about it during our 1:1 since geographical attributes being recognized depend on it being part of the measures.